run_tests_language

```diff
@@ -4,6 +4,7 @@ stty cols 80
 stty rows 20
 
 language=$1
+expect_path="$(echo $2 | sed 's;/;\\/;g')"
```

Comment:
```suggestion
expect_path=$2
```

Reply:
This is needed to escape the slashes in the path since we pass it in a bash command, I tried removing it locally and it started failing again. Will add a comment.

Reply:
Nevermind, adding double quotes in your other comment makes this work :smile: